### PR TITLE
Expose mutation testing report

### PR DESCRIPTION
**Pull request by thomaschaplin**
Fixes https://github.com/trayio/mock-inspect/issues/36

This PR exposes the mutation testing report via GitHub pages - https://trayio.github.io/mock-inspect/mutation/
The readme badge will need to be updated manually (there will be a new task to automate this)

### CODE REVIEW

**Comment by thomaschaplin** [comment]
As discussed with @rickschubert I shall look into moving this into a GitHub action as an artifact instead of living in the main repo. It can be run via a CRON - monthly maybe?

**Comment by rickschubert** [comment]
@thomaschaplin I think weekly would be fine even, or twice a week.

**Review by rickschubert — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ mock-inspect
 ![Function coverage](https://img.shields.io/endpoint?url=https%3A%2F%2F35fec4cb-1fbe-4aed-be1f-ea4814d9ba46.trayapp.io%2Ffunctions)
 ![Line coverage](https://img.shields.io/endpoint?url=https%3A%2F%2F35fec4cb-1fbe-4aed-be1f-ea4814d9ba46.trayapp.io%2Flines)
 ![Branches badge](https://img.shields.io/endpoint?url=https%3A%2F%2F35fec4cb-1fbe-4aed-be1f-ea4814d9ba46.trayapp.io%2Fbranches)
+[![Mutation badge](https://img.shields.io/badge/Mutation%20Coverage-88.93-green)](https://trayio.github.io/mock-inspect/reports/mutation/html/)
```
Comment (rickschubert) [inline README.md:10]:
```suggestion
[![Mutation badge](https://img.shields.io/badge/Mutation%20Coverage-88.93-green)](https://trayio.github.io/mock-inspect/mutation/)
```

**Review by rickschubert — CHANGES_REQUESTED** [review]
The path needs to change due to the rename

**Pull request merged**